tests/rds_instance.tf

```diff
@@ -13,6 +13,7 @@ module "rds_instance" {
   multi_az                    = true
   storage_type                = "gp2"
   allocated_storage           = "100"
+  storage_encrypted           = false
```

Comment:
Use strings "false" and "true" for boolean values, _e.g._ `storage_encrypted  = "false"`.
Terraform does not work with plain booleans correctly in many cases, and they recommend using strings.